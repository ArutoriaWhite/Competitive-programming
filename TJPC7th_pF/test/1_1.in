35282419 2
10825492 0
12930614 0
