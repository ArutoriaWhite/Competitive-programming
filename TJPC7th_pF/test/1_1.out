12930614 2105122
